app/lib/dfe/bigquery/application_metrics.rb

```diff
@@ -41,6 +42,22 @@ def self.candidate_headline_statistics(cycle_week:)
 
         new(result)
       end
+
+      def self.age_group(cycle_week:)
+        results = ::DfE::Bigquery.client.query(
+          <<~SQL,
+            SELECT *
+            FROM dataform.application_metrics
+            WHERE recruitment_cycle_year = #{RecruitmentCycle.current_year}
+            AND cycle_week = #{cycle_week}
+            AND subject_filter_category = "Total excluding Further Education"
+            AND nonsubject_filter_category = "Age group"
+            ORDER BY nonsubject_filter ASC
+          SQL
```

Comment:
I am still using raw SQL here. Let me know how you would prefer to tackle the two SQLs that we have on this class.
I thought to add a small DSL like:

```ruby
results = ::DfE::Bigquery.client.query(
  where(recruitment_cycle_year:, cycle_week:, subject_filter_category: 'Total excluding Further Education',  nonsubject_filter_category: 'Age group').order(nonsubject_filter: :asc).to_sql
)
```

But then I will have to create a small Arel methods which I am not sure if will pay the benefits. Let me know your thoughts!

Reply:
I would be interested in moving more towards this approach – The queries are almost identical and we will have six of these!

Reply:
done! I [didn't make work with ](https://ukgovernmentdfe.slack.com/archives/C05D5S8C6PJ/p1699542530358379)Arel::Table  so I had to create a small and simple approach